packages/scheduler/src/Scheduler.js

```diff
@@ -31,6 +31,9 @@ var IDLE_PRIORITY = maxSigned31BitInt;
 var firstCallbackNode = null;
 
 var currentDidTimeout = false;
+// Pausing the scheduler is useful for debugging.
+var isSchedulerPaused = false;
```

Comment:
I think I might want to expose this variable too, since for debugging it's necessary to know if the thing is paused at all in the first place.